src/platform/android/DnssdImpl.cpp

```diff
@@ -224,6 +224,7 @@ CHIP_ERROR ChipDnssdStopBrowse(intptr_t browseIdentifier)
         ChipLogError(Discovery, "Java exception in ChipDnssdStopBrowse");
         env->ExceptionDescribe();
         env->ExceptionClear();
+        chip::Platform::Delete(ctx);
```

Comment:
This seems ok, but wouldn't it be simpler to just delete it after the CallVoidMethod() before we do the ExceptionCheck() at all, since we are done with it at that point?

Either way, though.

Reply:
Right, at that point we used the callbackHandle in NsdManagerServiceBrowser stopDiscover() and returned so we don't need it anymore. Changed. Thanks!